src/OpenTelemetry.Instrumentation.AWSLambda/AWSLambdaWrapper.cs

```diff
@@ -160,7 +162,14 @@ public static Task<TResult> TraceAsync<TInput, TResult>(
             }
         }
 
-        var functionTags = AWSLambdaUtils.GetFunctionTags(input, context);
+        // No parallel invocation of the same lambda handler.
+        bool faasColdStart = isColdStart;
+        if (faasColdStart)
```

Comment:
The `if` can be removed, it's fine to unconditionally set it to false here. Then we also might not need the local variable if you change it after calculating functionTags

Reply:
Yes, we can do it. I removed the local variable and moved setting of the internal one after the calculation of functionTags.